app/views/content/life-as-a-teacher/explore-subjects/biology/_article.html.erb

```diff
@@ -0,0 +1,125 @@
+    large: true
+  ) %>
+  </section>
+
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Shape your pupils' futures</h2>
+      <p>Teaching biology opens the door for your pupils to pursue exciting careers. They could work in a diverse range of fields, from zoology and wildlife conservation, to medicine and dentistry.</p>
+      <p>By solving challenging problems, your pupils could make significant contributions to these impactful areas.</p>
+      <p>Your pupils will learn more from you than you might think.  As their teacher, you'll guide them though the skills they need to show their understanding of biology. You'll teach them valuable life skills such as critical thinking, decision making and communication.</p>
+  </section>
+
+  <section class="col col-720">
+    <%= render Content::QuoteComponent.new(
+    text: "For me, science is all about being inquisitive. I was always the kid in class who was thinking, 'I wonder how that works, or I wonder why that does that'. I recognise that so much in the kids I teach and being able to give them the tools to develop that curiosity and knowledge is genuinely exhilarating.",
+    name: "Holly, science teacher",
+    large: true
+  ) %>
+  </section>
+
+  <section class="col col-720">
+    <h2 class="heading--box-blue">What you'll teach</h2>
+    <p>You'll teach the <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-science-programmes-of-study/national-curriculum-in-england-science-programmes-of-study">national curriculum for biology</a>.</p>
+    <p>You could have the opportunity to teach or support physics and chemistry lessons throughout key stages 3 and 4.</p>
+    <p>When you teach 11 to 14 year olds (key stage 3), you'll cover themes from the 3 science disciplines of biology, chemistry and physics.</p>
+    <p>The biology themes are:</p>
+    <ul>
+      <li>structure and function of living organisms</li>
+      <li>material cycles and energy</li>
+      <li>interactions and interdependencies</li>
+      <li>genetics and evolution</li>
+    </ul>
+
+    <p>Themes you'll cover when you teach 14 to 16 year olds (key stage 4) will be more specific to biology, and include:</p>
+    <ul>
+      <li>cell biology</li>
+      <li>transport systems</li>
+      <li>health, disease and the development of medicines</li>
+      <li>coordination and control</li>
+      <li>photosynthesis</li>
+      <li>ecosystems</li>
+      <li>evolution, inheritance and variation</li>
+    </ul>
+    <p><a href="https://www.thenational.academy/teachers/curriculum/science-secondary/units">Explore what teaching a key stage 4 biology lesson would be like</a>.</p>
+    <p>Teaching pupils at different ages and abilities means you'll always be expanding your knowledge and approach to meet pupils' needs. Once you have qualified teacher status (QTS) you can also teach any other subject, as long as you can show you have enough expertise.</p>
```

Comment:
@Sarah-DfE Think this needs to be below the KS5 section?

Reply:
@gemmadallmandfe good spot! Shall move it down